src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -7,7 +7,10 @@
   {:margin-top -4})
 
 (def paragraph
-  {:margin-top spacing-between-blocks})
+  {:margin-top           spacing-between-blocks
+   :position             :relative
```

Comment:
I had the impression all RN components by default use `:position :relative`, so this seems unnecessary.